CNAME

```diff
@@ -1 +0,0 @@
-2012.fossasia.org
```

Comment:
Why did you remove the CNAME file ?

Reply:
It was creating some issue with rendering on GitHub pages, not really sure what.

Reply:
That might be because of some other issues

Reply:
Add back the file